.travis.yml

```diff
@@ -1,26 +1,26 @@
 language: php
 
-sudo: false
-
-cache:
-    directories:
-        - $HOME/.composer/cache
+env:
+    global:
+        - SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
+        - APP_ENV=test
+        - APP_DEBUG=0
 
 matrix:
     include:
         - php: 7.1
 
-env:
-    global:
-        - APP_ENV=test
-        - APP_DEBUG=0
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
```

Comment:
You can cache `.phpunit` too (if you use ` bin/phpunit`, if you don't you can `cache vendor/bin/.phpunit`).